### DISCUSSION TITLE
Fee

### DISCUSSION THREAD

**Discussion by OmerDadash** [IBAX-io/go-ibax#676] [Soru Cevap - Türkçe]
Fee ücreti ne kadar olacak ve transfer hızı nedir ??

**Comment by Yusufknk** [comment]
Cüzdandan cüzdana transferde hemen yansır. Verilen 10000 ibxc işlem ücretleri olarak kullanabilirsiniz. Fee ücretleri işlemden işleme değişiklik göstermektedir.
